src/pudl/analysis/ferc1_eia.py

```diff
@@ -881,7 +873,11 @@ def prettyify_best_matches(
             train_df,
             match_set=match_set,
         )
-
+    connects_ferc1_eia = (
+        Package.from_resource_ids()
+        .get_resource("ferc1_eia")
+        .enforce_schema(connects_ferc1_eia)
+    )
```

Comment:
I think this can be done with just the Resource rather than the Package:

```py
Resource.from_id("ferc1_eia").enforce_schema(connects_ferc1_eia)
```